docs/Deploying-Pega-on-AKS.md

```diff
@@ -360,12 +360,13 @@ To customize these files, you must download them from the source github reposito
 
 ```bash
   $ helm search repo pega
-  NAME        CHART VERSION   APP VERSION     DESCRIPTION
-  pega/pega   1.2.0                           Pega installation on kubernetes
-  pega/addons 1.2.0           1.0             A Helm chart for Kubernetes
+  NAME              CHART VERSION   APP VERSION     DESCRIPTION
+  pega/pega             1.2.0                           Pega installation on kubernetes
+  pega/addons           1.2.0           1.0             A Helm chart for Kubernetes
+  pega/backingservices  1.2.0                           Helm Chart to provisioning backing services for Pega (eg. Search and Reporting Service)
```

Comment:
Helm Chart to provision the latest Search and Reporting Service for your Pega Infinity deployment.